### ISSUE TITLE
Remove experimental flag from stable APIs

### ISSUE THREAD

**Issue by ocombe** [angular/angular#21661]
A lot of our APIs stay flagged as "experimental" because we never check if we should change it.
Before each major release, we should ask each component owner to remove the experimental flag from APIs that have been around for more than 1 version.
For example almost everything in i18n is flagged experimental.

@StephenFluin @IgorMinar what do you think about that?

Task breakdown:

- [ ] remove @experimental and @stable from source code
- [ ] update API golden files
- [ ] remove stable and experimental badges from API templates
- [ ] remove "experimental" from the API filter

**Comment by pkozlowski-opensource** [comment]
> Before each major release, we should ask each component owner to remove the experimental flag from APIs that have been around for more than 1 version.

I would rephrase it as: "ask each component owner to review the existing experimental flags and remove this flag if API is considered stable". I'm not sure if "being around for more than 1 release" is a good indicator of API's stability...

**Comment by ocombe** [comment]
You're right :)

**Comment by IgorMinar** [comment]
I think we'll end up removing all stable/experimental labels. In reality since we want to make upgrades smooth, we can't make painful breaking changes to any of our commonly used apis, which includes many of the apis currently parked as experimental.

I don't know if we can do this still in the v6 time (it's kind of late for adding that to the backlog) or more likely in v7.

**Comment by jenniferfell** [comment]
@jbogarthyde FYI. To keep in mind as you work on API docs.

**Comment by jenniferfell** [comment]
@IgorMinar Can we close this now?

**Comment by jenniferfell** [comment]
animations is still tagged @experimental but is not
steps:
* @IgorMinar remove @experimental tag
* @petebacondarwin remove experimental from drop-down menu on API list/index

**Comment by jenniferfell** [comment]
not showing in jira. uncertain why

**Comment by petebacondarwin** [comment]
@IgorMinar and @mhevery  agreed that it would be better, instead of just removing all of the experimental jsdocs tags, to convert them into `@publicApi` or something like that. Essentially we will merge `@experimental` & `@stable` tags into one tag  `@publicApi`, indicating that an export is public, with deprecated symbols being additionally marked as `@deprecated`.

The goal is that during code review we know by looking at a symbol declaration that it’s a public api and it should have solid api doc.

**Comment by petebacondarwin** [comment]
One downside of this approach is identifying items that have been tagged with `@publicApi` but are not actually exported... and so not actually public 🤔
